README.md

```diff
@@ -6,9 +6,13 @@ A Momento API Token is required. You can generate one using the [Momento Console
 
 The Momento Cache module uses [Momento's PHP SDK](https://docs.momentohq.com/cache/develop/sdks/php) internally. While installing the Drupal module will automatically install the SDK for you, you will need to install and enable [the PHP gRPC extension](https://github.com/grpc/grpc/blob/master/src/php/README.md) separately for the SDK to function.
 
+### Drupal and Momento Rate Limiting
+
+Momento's free tier limits accounts' transactions per second (TPS) and throughput (KiBps), and requests that exceed those limits are throttled. Drupal's usage of the cache backend is significantly higher than the default limits, so it will be necessary to reach out to `support@momentohq.com` to raise your account limits to use Momento Cache as the default Drupal cache backend.
```

Comment:
s/it will be necessary/prior to using the integration, please reach out to ... ?

Also do you know how the error will manifest if they haven't raised their limits? If so, including that here would be useful.

Also - do you know what the numbers are? I wouldn't expect a sustained rate of momento requests that exceeded 100 tps tbh. I could imagine a large burst of them when the system is first loading and then would guess that it would drop down by quite a bit. Is that what you observed? do you have numbers? it would be good to be able to tell people what they should request the limits to be.

Lastly - if it is just the initial burst that causes the throttling, then adding some retries with a little backoff might get users through that burst and then allow things to work properly under the default throttling limit?

Reply:
Wording change is made, thanks. I don't have a great idea of what the numbers are, but defaults for rate and throughput were definitely not cutting it. It's not that there's an initial burst-- there's a burst with every page request. We'll get much better data from sites running at scale as they use the integration, and we can definitely do some more digging into the numbers for my minimal installation.